src/main/java/com/bhaskar/urlshortner/model/product/Product.java

```diff
@@ -0,0 +1,26 @@
+package com.bhaskar.urlshortner.model.product;
+
+import com.bhaskar.urlshortner.model.common.Category;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+
+@Entity
+@Data
+@Builder
+@NoArgsConstructor
```

Comment:
@Data already contain allargconstructor and noarg constructor  why we explicitlyadding it